code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -456,7 +456,7 @@
 /obj/item/modular_computer/Exited(atom/movable/gone, direction)
 	if(internal_cell == gone)
 		internal_cell = null
-		if(enabled && !use_power())
+		if(enabled && !use_energy())
```

Comment:
`use_energy()` with no arguments? This should take some positive value right? Same question for everywhere else in this file

Reply:
I guess they thought use_power() meant something else. What a strange use case. I'll add arguments or remove these once I find out what's going on in there.

Reply:
Seems like they have their own proc. Not really relevant to ours and it just got caught up in the mass proc name change. I guess it doesn't matter, though it's confusing. Maybe for a separate PR.